pkg/apis/servicecatalog/register.go

```diff
@@ -56,6 +56,13 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&metav1.GetOptions{},
 
 		&Broker{},
```

Comment:
Yeah, that's correct.  Those types have to be 'known' to the API machinery for this group so that they can be converted.  Make sense?